docs/modules/deploy/pages/deploying-on-aws.adoc

```diff
@@ -92,9 +92,108 @@ The following optional properties can be configured[[ec2_properties]]:
 
 Note that if you don't specify any of the properties, then Hazelcast uses the IAM Role assigned to the EC2 Instance to form a cluster from all Hazelcast members running in same region.
 
-=== EC2 Hazelcast Client Configuration
+=== Client Discovery of EC2 Hazelcast Cluster
```

Comment:
Is it possible to list all topologies at the side listing?

<img width="257" alt="Screenshot 2023-07-17 at 17 01 20" src="https://github.com/hazelcast/hz-docs/assets/6005622/08805b2f-dc7b-4849-843d-d5c28bceccb2">

Reply:
As far as I can understand, only L2 titles (`==`) are listed there

Reply:
I see, @Serdaro can we change this rule for this specific page?

Reply:
Do you have any idea why AWS page does not list them?